ansible/README.md

```diff
@@ -306,14 +309,15 @@ https://www.enterprisedb.com/postgres-tutorials/postgresql-replication-and-autom
 ### Useful ansible commands:
 
 ```terminal
-ansible-inventory -i inventory.yml --graph
+uv run ansible-inventory -i inventory.yml --graph
 
-ansible servers -m ping -u ubuntu -i inventory.yml
+uv run ansible servers -m ping -u ubuntu -i inventory.yml
```

Comment:
Its actually not servers anymore but rather staging or production
`uv run ansible production -m ping -u ubuntu -i inventory.yml`
but then I get the error: ERROR! Attempting to decrypt but no vault secrets found
So maybe I need to store the secret locally somehow...

Reply:
> is this warning a future issue?
> 
> ```
> [WARNING]: Platform linux on host lard-b is using the discovered Python interpreter at /usr/bin/python3.12, but future installation of another Python interpreter could change the meaning of that path.
> See https://docs.ansible.com/ansible-core/2.17/reference_appendices/interpreter_discovery.html for more information.
> ```

uv solves this, it’s using the Python specified in our pyproject.toml. Ansible just isn’t smart enough to know that so it warns anyway